index.js

```diff
@@ -1,102 +1,158 @@
-//         twoNumber = +(prompt('введіть друге число'));
-//         alert(`${oneNumber} + ${twoNumber} = ${oneNumber + twoNumber}`);
-//         break;
-
-//     case "sub":
-//         oneNumber = +(prompt('введіть перше число'));
-//         twoNumber = +(prompt('введіть друге число'));
-//         alert(`${oneNumber} - ${twoNumber} = ${oneNumber - twoNumber}`);
-//         break;
-
-//     case "mult":
-//         oneNumber = +(prompt('введіть перше число'));
-//         twoNumber = +(prompt('введіть друге число'));
-//         alert(`${oneNumber} * ${twoNumber} = ${oneNumber * twoNumber}`);;
-//         break;
-
-//     case "div":
-//         oneNumber = +(prompt('введіть перше число'));
-//         twoNumber = +(prompt('введіть друге число'));
-//         alert(`${oneNumber} / ${twoNumber} = ${oneNumber / twoNumber}`);;
-//         break;
-        
-//     default:
-//         alert("Читай умову!");
-//         break;
-// };
-
-// let firstNumber = +prompt('введіть перше число');
-// let secondNumber = +prompt('введіть друге число');
-// let add = `${firstNumber}+${secondNumber} = ${firstNumber + secondNumber}`;
-// let sub = `${firstNumber}-${secondNumber} = ${firstNumber - secondNumber}`;
-// let mult = `${firstNumber}*${secondNumber} = ${firstNumber * secondNumber}`;
-// let div = `${firstNumber}/${secondNumber} = ${firstNumber / secondNumber}`;
-// let resolt = `Ваш результат:\n${add}\n${sub}\n${mult}\n${div}`;
-// alert(resolt);
-
-
-//homework2/1
-
-// let surName = prompt ('Enter your last name');
-
-// let firstName = prompt('Enter your firs name');
-
-// let patronymic = prompt('Enter your middle name');
-
-// let greeting = 'good morning ' + surName + ' ' + firstName + ' ' + patronymic;
-// // let greeting = `Good Morning ${surName} ${firstName} ${patronymic}`;
-
-
-// alert(greeting);
-
-
-
-
-// HomeWork2/2
-
-// let number =  92358;
-// let string = String(number);
-// let newString = '';
-// for(i = 0; i < string.length ; i++){
-//     newString += `${string[i]} `;
-// };
-// console.log(newString);
-
+// Дан масив [16,-37,54,-4,72,-56,47,4, -16,25,-37,46,4,-51,27,-63,4,-54,76,-4,12,-35,4,47]
+// Знайти суму та кількість позитивних елементів.
+
+const arrNumbers = [
+    16, -37, 54, -4, 72, -56, 47, 4, -16, 25, -37, 46, 4, -51, 27, -63, 4, -54,
+    76, -4, 12, -35, 4, 47,
+];
+let amountPositiveElements = 0;
+const arrNumbersSum = arrNumbers.reduce((accum, item) => {
+    if (item > 0) {
+        return accum + item;
+        amountPositiveElements++;
+    }
+    return accum;
+}, 0);
+console.log(arrNumbersSum);
+console.log(amountPositiveElements);
+
+// Знайти мінімальний елемент масиву та його порядковий номер.
+
+let arrNumbers1 = [
+    16, -37, 54, -4, 72, -56, 47, 4, -16, 25, -37, 46, 4, -51, 27, -63, 4, -54,
+    76, -4, 12, -35, 4, 47,
+];
+let sequenceNumberMinEl;
+let minArrayElement = arrNumbers1.reduce((accum, item, index) => {
+    if (item < accum) {
+        return accum = item;
```

Comment:
не треба змінювати параметр функції, просто повертайте значення:

```suggestion
        return item;
```